web/src/components/TaskCtaArea.tsx

```diff
@@ -0,0 +1,123 @@
+import { PrimaryButtonDropdown } from "@/components/njwds-extended/PrimaryButtonDropdown";
+import { TaskCTA } from "@/components/TaskCTA";
+import { useUserData } from "@/lib/data-hooks/useUserData";
+import { PostOnboarding, Task } from "@/lib/types/types";
+import { openInNewTab } from "@/lib/utils/helpers";
+import { ProfileData } from "@businessnjgovnavigator/shared/index";
+import { useRouter } from "next/router";
+import React, { ReactElement } from "react";
+
+interface ButtonsArrayElement {
```

Comment:
[sand] I can't think of a good name right now, but I feel like there's a better name out there for this. I feeling like using the term `ArrayElement` in a type name is an antipattern because it's describing _how_ the type is being used instead of _what_ it is/represents.

Reply:
You're right, naming is tricky, because it defines an element that can either be a button or a dropdown list item depending on how many elements in the array.